src/moonbeam/moonbase-local.js

```diff
@@ -165,7 +165,7 @@ const main = async () => {
     const turingAddressETH = aliceKeyPair.address;
     console.log('Turing wallet’s Ethereum address:', turingAddressETH);
 
-    const topUpAmount = (new BN(1000).mul(decimalBN)).toString();
+    const topUpAmount = (new BN(1000000).mul(decimalBN)).toString();
```

Comment:
What is the reason for the increment? We can’t create a proxy with 1000 DEV any more?